.github/workflows/pre-main.yaml

```diff
@@ -160,3 +160,53 @@ jobs:
       - name: (if on main and upstream) Push the newly built image to Quay.io
         if: ${{ github.ref == 'refs/heads/main' && github.repository_owner == 'test-network-function' }}
         run: docker push --all-tags ${REGISTRY}/${IMAGE_NAME}
+ 
+  catalog:
+    name: Update catalog
+    needs: [lint, unit-tests, smoke-tests]
+    runs-on: ubuntu-20.04
+    env:
+      SHELL: /bin/bash
+      KUBECONFIG: '/home/runner/.kube/config'
```

Comment:
Not needed?

Reply:
Thanks, KUBECONFIG is not needed, will remove.